Testing/BasicFunctionsExample.py

```diff
@@ -0,0 +1,59 @@
+import PyBoolNet
```

Comment:
Could you move the MPBN demonstration to its own file and revert the changes here? I'm not 100% sure, but I think we reference the contents of this file in some documentation.

Reply:
I left the original BasicFunctionsExample.py untouched in the Example and Tutorials folder.
I renamed the copied file in Testing folder to MPBN_BasicFunctionsTesting